### PR TITLE
Hungarian instance domain update

### PR DESCRIPTION
**Pull request by pragai**

### CODE REVIEW

File: inventory/host_vars/openfoodnetwork.hu/config.yml
Comment (mkllnk) [inline inventory/host_vars/openfoodnetwork.hu/config.yml:1]: You have to rename the name of the folder as well.

**Review by mkllnk — CHANGES_REQUESTED** [review]

**Comment by pragai** [comment]
Created requested changes, renamed folder.

**Review by mkllnk — APPROVED** [review]
This is looking good to me.

**Review by jibees — APPROVED** [review]
I'm not that comfortable with this part of the code, but looks good to me as well. 

Wondering what are the next steps? Should we synchronize the merging/provisioning of this PR to another task?

**Comment by mkllnk** [comment]
> Wondering what are the next steps? Should we synchronize the merging/provisioning of this PR to another task?

Hungary is in the managed instances. So we should provision this. But maybe good to arrange a date with @pragai. When would it suit you to switch domains?

**Comment by pragai** [comment]
Hi, please do the merge whenever suits you. We are in the kickoff process of boarding the instance, so it is a good time to change the domain now. Thanks!

**Comment by mkllnk** [comment]
Okay, it's merged. Shall we proceed with provisioning or are you doing that?

**Comment by pragai** [comment]
I!ll gladly do that.  Is it simply: ansible-playbook playbooks/provision.yml --limit=hu-prod ?

**Comment by mkllnk** [comment]
Almost. You just need the secrets as well. Hm, I'm just realising that we added the n8n and Metabase integration to your instance via the secrets. If you don't need those then you can just go ahead. Otherwise, I need to share the new file with you or I just do the provisioning.

**Pull request merged**